capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -120,18 +120,20 @@ function createAutocompleteListElement(collegeName, value) {
 }
 
 /**
- * Adds valid suggestions to list of stored suggestions
- * @param {string} validSuggestion
+ * Keeps track of chosen colleges and the return value is for testing
```

Comment:
nit - you've refactored this s.t. the return value is used in code. update the comment to reflect this.

Reply:
done!